### DISCUSSION TITLE
Difference between Bitbucket and Bitbucket Server

### DISCUSSION THREAD

**Discussion by septatrix** [gitkraken/vscode-gitlens#1386] [Feedback]
What is the difference between these two remote providers? My initial guess is that one should be used for self-hosted instances however GitLab which can also be self hosted does not have such a differentiation

**Comment by eamodio** [comment]
Your guess is correct, but unlike GitHub/GitLab the self-hosted version of BitBucket isn't really the same product as the hosted version, so there are some differences in the url structures between the two. Whereas GitHub/GitLab have the same url structures for hosted vs self-hosted.

**Reply by wrightleft** [reply]
So--which one is for the self-hosted instance?
